utils/guidelines_xslt/odd2html/guidelines.xsl

```diff
@@ -400,11 +400,56 @@
     
     <xd:doc>
         <xd:desc>
-            <xd:p>term</xd:p>
+            <xd:p>Names</xd:p>
+        </xd:desc>
+    </xd:doc>
+    <xsl:template match="tei:name" mode="guidelines">
+        <xsl:choose>
+            <xsl:when test="not(@ref)">
+                <xsl:apply-templates select="node()" mode="#current" />
+            </xsl:when>
+            <xsl:otherwise>
+                <a class="link_ref link_external" href="{@ref}">
+                    <xsl:apply-templates select="node()" mode="#current" />
+                </a>
+            </xsl:otherwise>
+        </xsl:choose>
+    </xsl:template>
+    
+    <xd:doc>
+        <xd:desc>
+            <xd:p>Terms</xd:p>
         </xd:desc>
     </xd:doc>
     <xsl:template match="tei:term" mode="guidelines">
-        <span class="term"><xsl:apply-templates select="node()" mode="#current"/></span>
+        <span class="term">
+            <xsl:choose>
+                <xsl:when test="not(@ref)">
+                    <xsl:apply-templates select="node()" mode="#current" />
+                </xsl:when>
+                <xsl:when test="starts-with(@ref,'#')">
+                    <xsl:variable name="chapter.id" select="substring(@target,2)" as="xs:string" />
+                    <xsl:variable name="tocInfo" select="$all.chapters/descendant-or-self::chapter[@xml:id = $chapter.id]" as="node()?" />
+                    <xsl:choose>
+                        <xsl:when test="exists($tocInfo)">
+                            <a class="link_ref chapterLink" title="{$tocInfo/@number || ' ' || $tocInfo/@head}" href="#{$chapter.id}">
+                                <xsl:apply-templates select="node()" mode="#current" />
+                            </a>
+                        </xsl:when>
+                        <xsl:otherwise>
+                            <span class="wrong_ref" data-target="{$chapter.id}">
+                                <xsl:apply-templates select="node()" mode="#current" />
+                            </span>
+                        </xsl:otherwise>
```

Comment:
I have a question on this

Reply:
it basically says that every internal link (starting with a hash, according to line +430) needs to point to a chapter, or it is labelled a `wrong_ref`. I haven't looked up if there are any such cases, but it feels a bit too opinionated. Probably, when pointing to elements _as musical concepts_ (like _tempo_), we would still encourage to use `<gi>` instead, but I wonder if we should already label every other link target as wrong. I would probably prefer a more neutral `unspecified_ref` or some such…

Reply:
@kepper Actually this `wrong_ref ` wasn't my idea, but was simply copied over. I see your point, but this is not really something that is connected to the PR and should be addressed in another PR.

Reply:
Thanks for the clarification. Then let's go ahead!